lib/screens/registration/enter-phone.dart

```diff
@@ -0,0 +1,185 @@
+import 'dart:convert';
+
+import 'package:bc_app/constants/country_codes.dart';
+import 'package:flutter/material.dart';
+
+class EnterPhone extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _EnterPhone();
+}
+
+class _EnterPhone extends State<EnterPhone> {
+  static String dropdownValue = 'IN';
+  static String _phoneNumber;
+  static String isdCode = '+91';
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        appBar: AppBar(),
+        body: getBody(),
+        floatingActionButton: Padding(
+          padding: const EdgeInsets.only(right: 16),
+          child: FloatingActionButton(
+            child: new Icon(
+              Icons.navigate_next,
+              size: 48,
+            ),
+            backgroundColor: Color(0xff8FC7E8),
+          ),
+        ),
+      ),
+    );
+  }
+
+  getBody() {
+    return Padding(
+        padding: EdgeInsets.symmetric(horizontal: 12),
+        child: Padding(
+          padding: const EdgeInsets.all(24),
+          child: Form(
+              child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: getBodyContent(),
+          )),
+        ));
+  }
+
+  List<Widget> getBodyContent() {
+    List<Widget> widgets = [];
+
+    widgets.add(getTitle());
+    widgets.add(getSubTitle());
+    widgets.add(getisdCodeInfo());
+    widgets.add(getPhoneNumber());
+    widgets.add(getToU());
+
+    return widgets;
+  }
+
+  Widget getTitle() {
+    return Padding(
+      padding: const EdgeInsets.only(right: 128),
+      child: Text(
+        'Enter your phone number',
+        style: TextStyle(
+            fontSize: 26,
+            letterSpacing: 0.17,
+            color: Color(0xff8FC7E8),
+            fontFamily: 'Roboto'),
```

Comment:
add a comma at the end for better dart formatting.

```suggestion
            fontFamily: 'Roboto',),
```